Scripts/Editor/AssemblyInjections/UnityEditor/ConsoleHelper.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.Reflection;
+using UnityEditor;
+using UnityEngine;
+
+namespace Anvil.Unity.Editor.Debug
+{
+    public static class ConsoleHelper
+    {
+        // TODO: Future
+        // Should one wish to extend this functionality, adding this as a custom button overlaid on the Console Window
+        // is probably helpful and better than a tools menu.
+        // Additional features would be the ability to keep track of which error we are on and be able to go next/back
+        // as well as first and last. 
+        
+        //Had to figure this out from old Unity decompiled code. Not sure where this is defined today sadly.
+        //But it's unlikely to ever change as so much would have to be changed in the Unity Editor to accomodate.
+        private const int ERROR_FLAG = 512;
+        
+        [MenuItem("Anvil/Debug/Skip to First Error")]
+        internal static void SkipToFirstError()
```

Comment:
A skip to next error would be nice as well but we can do later if it's not a super quick addition.
Often when compiling+running you get some native dispose errors from the previous run instance's crash. In those cases the first error(s) are unrelated to a runtime issue.

Reply:
Yeah it'll have to be later, it's not super difficult I don't think, but will require a bit of effort.